package.json

```diff
@@ -23,6 +23,7 @@
     "prop-types": "^15.5.8",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
+    "react-element-proptypes": "^1.0.0",
```

Comment:
There was [an issue in the `React` repo that asked for element-specific `PropType` checking](https://github.com/facebook/react/pull/7119). 

Dan Abramov had [this to say](https://github.com/facebook/react/pull/7119#issuecomment-228615386)
> My personal opinion is that requiring elements to be of a specific type is an antipattern more often than not. I’m not saying it never happens, but it encourages tightly coupling components and makes it harder to introduce “transparent” wrappers around children.

> So while there may be use cases for this, I would say we’d rather not encourage this pattern, and it would better live as a third party library. Additionally we are very hesitant to expand the surface API area of PropTypes which we plan to eventually extract from React.

So why do I think it would be useful in [this use-case](https://github.com/edx/paragon/pull/62/files#diff-3a3b7078694a1e5b19d02a0bd3dcc9c2R172)? 

Because we actually want to encourage tight coupling - a `RadioButtonGroup` should not have children that aren't `RadioButton`s.